extension/src/documents.ts

```diff
@@ -103,15 +103,29 @@ export class DocumentManager{
 		}
 
 		// This builds the '_workspace' *.LSP Memory Document placeholder set
-		// 		This feature does make the "fair assumption" that AutoCAD machines has plenty of memory to be holding all this information
-		// 		This was stress tested with a root workspace folder containing 10mb of *.LSP files and the memory footprint increased less than 50mb
+		// 		This feature does make the "fair assumption" that AutoCAD machines have plenty of memory to be holding all this information
+		// 		The impact of creating read-only documents was stress tested with a root workspace folder containing 10mb of *.LSP files
+		//		and the memory footprint from the readonlyDocument's increased the memory (sustained) by less than 50mb		
 		vscode.workspace.findFiles("**").then((items: vscode.Uri[]) => {
 			items.forEach((fileUri: vscode.Uri) => {			
 				if (this.getSelectorType(fileUri.fsPath) === DocumentManager.Selectors.lsp) {
-					this._workspace.set(fileUri.fsPath, null); // initialized as null to blueprint the workspace files without the overhead of reading their contents
+					this._workspace.set(fileUri.fsPath, ReadonlyDocument.open(fileUri.fsPath));
```

Comment:
This is the only pre-load happening as of pre-check method (17a9977) and for me this is 4-10 seconds on a 10mb of LSP's.

Reply:
I also didn't get slow feeling to open a workspace which contains lots of files